src/react.js

```diff
@@ -10,7 +10,7 @@ module.exports = {
     'plugin:testing-library/react',
     'plugin:cypress/recommended',
   ],
-  plugins: ['react', 'testing-library'],
+  plugins: ['react', 'testing-library', 'unicorn'],
```

Comment:
Since the plugin was imported `index.js`, we may don't need to re-declare it here